CMakeLists.txt

```diff
@@ -289,6 +292,21 @@ if(NRN_ENABLE_CORENEURON)
                             ${CMAKE_BINARY_DIR}/hh.mod.2
                     COMMENT "Update hh.mod for CoreNEURON compatibility")
   add_dependencies(nrniv hh_update)
+else()
+  if(EXISTS "${CMAKE_SOURCE_DIR}/src/nrnoc/hh.mod.orig")
+    add_custom_target(hh_restore
+            COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/src/nrnoc/hh.mod.orig
+            ${CMAKE_SOURCE_DIR}/src/nrnoc/hh.mod
+            COMMAND -E remove ${CMAKE_SOURCE_DIR}/src/nrnoc/hh.mod.orig
+            COMMENT "Restore hh.mod for NEURON")
+    add_dependencies(nrniv hh_restore)
+  else()
+    add_custom_target(hh_orig
+            COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/src/nrnoc/hh.mod
+                    ${CMAKE_SOURCE_DIR}/src/nrnoc/hh.mod.orig
+            COMMENT "Copy hh.mod for NEURON")
+    add_dependencies(nrniv hh_orig)
+  endif()
```

Comment:
In the future, we can copy *.mod files in the build directory and use those during build instead of running nocmodl in src directory directly.